cli/docs/reference/ocm_add_component-version.md

```diff
@@ -71,9 +71,15 @@ add component-version --repository oci::http://localhost:8080/my-repo --construc
                                                       (must be one of [abort-and-fail replace skip]) (default abort-and-fail)
       --concurrency-limit int                         maximum number of component versions that can be constructed concurrently. (default 4)
   -c, --constructor path                              path to the component constructor file (default component-constructor.yaml)
+      --display-mode enum                             display mode can be used in combination with --recursive
+                                                        static: print the output once the complete component graph is discovered
+                                                        live (experimental): continuously updates the output to represent the current discovery state of the component graph
```

Comment:
I think this should be updated with "continuously updates the output to represent the current construction state of the component graph". WDYT?